crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -933,23 +771,8 @@ fn handle_module_level_own_line_comment_before_class_or_function_comment<'a>(
 fn handle_slice_comments<'a>(
     comment: DecoratedComment<'a>,
     locator: &Locator,
+    expr_slice: &'a ExprSlice,
```

Comment:
Nit (applies to all handle functions): I would expect `expr_slice` to come before `locator` because it is, together with the comment, the main data the method operates on.